pkg/kor/create_test_resources.go

```diff
@@ -321,4 +321,33 @@ func CreateTestReplicaSet(namespace, name string, specReplicas *int32, status *a
 		},
 		Status: *status,
 	}
-}
\ No newline at end of file
+}
+
+func CreateTestDaemonSet(namespace, name string, labels map[string]string, status *appsv1.DaemonSetStatus) *appsv1.DaemonSet {
+	return &appsv1.DaemonSet{
+		ObjectMeta: v1.ObjectMeta{
+			Namespace: namespace,
+			Name:      name,
+			Labels:    labels,
+		},
+		Spec: appsv1.DaemonSetSpec{
```

Comment:
nit: do we need the spec part of the daemonset?

Reply:
Was indeed excessive, test ran successfully without it. Removed.
Thanks